sim/core/apl_values_aura.go

```diff
@@ -7,13 +7,39 @@ import (
 	"github.com/wowsims/sod/sim/core/proto"
 )
 
+type APLValueAuraIsKnown struct {
+	DefaultAPLValueImpl
+	aura AuraReference
+}
+
+func (rot *APLRotation) newValueAuraIsKnown(config *proto.APLValueAuraIsKnown) APLValue {
+	aura := rot.GetAPLAura(rot.GetSourceUnit(config.SourceUnit), config.AuraId)
+	return &APLValueAuraIsKnown{
+		aura: aura,
+	}
+}
+func (value *APLValueAuraIsKnown) Type() proto.APLValueType {
+	return proto.APLValueType_ValueTypeBool
+}
+func (value *APLValueAuraIsKnown) GetBool(sim *Simulation) bool {
+	return value.aura.Get() != nil
+}
+func (value *APLValueAuraIsKnown) String() string {
+	return fmt.Sprintf("Aura Active(%s)", value.aura.String())
+}
+
 type APLValueAuraIsActive struct {
 	DefaultAPLValueImpl
 	aura AuraReference
 }
 
 func (rot *APLRotation) newValueAuraIsActive(config *proto.APLValueAuraIsActive) APLValue {
 	aura := rot.GetAPLAura(rot.GetSourceUnit(config.SourceUnit), config.AuraId)
+
+	if aura.Get() == nil {
+		return nil
+	}
```

Comment:
These two additions are reverting a previous change. This brings `AuraIsActive` back to how it exists in Wrath and Cata as well